README.md

```diff
@@ -10,4 +10,173 @@ The HTML version of the ontology documentation is hosted at our
+
+To know which attributes are required, what type of values are supported and
+other information about the identity, we define `vocabularies` for each of the
+contexts. The vocabularies can be found under the [vocabularies](vocabularies)
+folder.
+
+The context defines 
+`@vocab: https://platformoftrust.github.io/standards/vocabularies/apartment.jsonld#`
+from where the information can be found.
+
+The `supportedClass` is a list of supported types for the identity in question.
+Usually there's only one supportedClass per identity. 
+
+The type also defines a list of `supportedAttribute` which
+defines each of the attributes the identity can have. All attribute objects
+MUST have `@type: pot:SupportedAttribute`.
+
+Below you can see an example of the apartment attributes defined in the 
+apartment vocabulary.
+
+    ...
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:name",
+      "dli:title": "Name",
+      "dli:description": "The name of the apartment.",
+      "dli:required": true
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "dli:data",
+      "dli:title": "Data",
+      "dli:description": "Additional key-value data to be saved for the apartment.",
+      "dli:required": true,
+      "dli:valueType": "xsd:object"
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:description",
+      "dli:title": "Description",
+      "dli:description": "Description of the apartment.",
+      "dli:required": false
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:completionYear",
+      "dli:title": "Completion year",
+      "dli:description": "The year when the apartment was completed",
+      "dli:required": false
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:inaugurationYear",
+      "dli:title": "Inauguration year",
+      "dli:description": "The year when the apartment was inaugurated",
+      "dli:required": false
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:height",
+      "dli:title": "Height",
+      "dli:description": "The height of the apartment",
+      "dli:required": false
+    },
+    {
+      "@type": "pot:SupportedAttribute",
+      "dli:attribute": "pot:usageType",
+      "dli:title": "Usage type",
+      "dli:description": "What is the apartment used for",
+      "dli:required": false
+    },
+    ...
+
+As can be seen from the example, we define the attribute, the title for the
+attribute, description and if the attribute is required or not. We can also
+define the value type of the attribute, e.g. `"dli:valueType": "xsd:object"`.
+If no `valueType` is defined, the default should always be `xsd:string`.
+We can also define default values for the attributes with 
+`"dli:defaultValue": "Placeholder"`,
+as well as different supported values with 
+`"dli:supportedValue": ["Yes", "No", "None"]`.
```

Comment:
Maybe `values` in plural?

Reply:
As with the `supportedClass` and `supportedAttribute`, this is in singular. It's definition in the ontology says what it is, and what it means. I think it's best to stick to singular, as the value can either be a list, or just a single string value.